README.md

```diff
@@ -197,59 +197,83 @@ hits.
 The visual tests use jest-image-snapshot to compare screenshots of the components to their expected state. This
 setup is configured for an exact visual match, so the tests must be run on a consistent platform. The browser used for
 the tests is downloaded via the puppeteer npm package. However, it is also important to run these tests on a consistent
-operating system. The cleanest way to do this is via docker - specifically, to run the tests in a docker environment
-identical to the one which the CI system uses. This environment is defined in the `Dockerfile` present within the
-the top level directory of the repository. To build the docker image, run the following command from this directory:
+operating system. The cleanest way to do this is via Docker - specifically, to run the tests in a Docker environment
+identical to the one which the CI system uses.
+
+To test in a Docker environment, first [install Docker on your computer](https://docs.docker.com/get-docker/). Then,
+within the top of the directory, log into Docker by running the following command. You may be asked to input
```

Comment:
"top of the directory" doesn't really make sense to me.  I think the original wording "top level directory of the repository" was better

Reply:
Fixed [952550d](https://github.com/sonatype/sonatype-react-shared-components/pull/738/commits/952550d70ab74e3b45a83deda7e9d4e5f17eedde)